pages/index.js

```diff
@@ -1,102 +1,174 @@
     marginTop: theme.spacing(1),
   },
   submit: {
     margin: theme.spacing(3, 0, 2),
-  }
-
+  },
 }));
 
 const theme = createMuiTheme({
   overrides: {
     MuiButton: {
       root: {
-        background: 'linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)',
+        background: "linear-gradient(45deg, #FE6B8B 30%, #FF8E53 90%)",
         borderRadius: 3,
         border: 0,
-        color: 'white',
+        color: "white",
         height: 48,
-        padding: '0 30px',
-        boxShadow: '0 3px 5px 2px rgba(255, 105, 135, .3)',
+        padding: "0 30px",
+        boxShadow: "0 3px 5px 2px rgba(255, 105, 135, .3)",
       },
     },
     MuiCheckbox: {
       root: {
-        padding: '1px 9px'
-      }
-    }
+        padding: "1px 9px",
+      },
+    },
   },
 });
 
-const MyFormTemplate = (props) => <FormTemplate {...props} showFormControls={false} />
+const MyFormTemplate = (props) => (
+  <FormTemplate {...props} showFormControls={false} />
+);
 
 const validate = (values) => {
-  console.log(values)
+  console.log(values);
   const errors = {};
 
-  for(let i=1; i<=17; i++){
-    if (values.SDGs && values.SDGs.includes(i) && ! values['evidenceText'+i] && ! values['evidenceURL'+i]) {
-      errors['evidenceText'+i] = 'Either the description or a URL is required';
-      errors['evidenceURL'+i] = 'Either the description or a URL is required';
+  for (let i = 1; i <= 17; i++) {
+    if (
+      values.SDGs &&
+      values.SDGs.includes(i) &&
+      !values["evidenceText" + i] &&
+      !values["evidenceURL" + i]
+    ) {
+      errors["evidenceText" + i] =
+        "Either the description or a URL is required";
+      errors["evidenceURL" + i] = "Either the description or a URL is required";
     }
   }
 
   return errors;
+};
+
+function openPR(values) {
+  let sdgNumber, evidenceText;
+  for (let i = 0; i < values.SDGs.length; i++) {
+    sdgNumber = JSON.stringify(values.SDGs[i]);
+    sdgNumber = parseInt(sdgNumber);
+    evidenceText = "evidenceText".concat(sdgNumber); //
+
+    values.SDGs[i] = {
+      SDGNumber: sdgNumber,
+      evidenceText: values[evidenceText],
+    };
+
+    delete values[evidenceText];
+  }
+
+  const myJSON = JSON.stringify(values, null, "\t");
+
+  // Return project json file aligned with naming convention
+  let name = values.name.toLowerCase().replace(/ /g, "_") + ".json";
+
+  const MyOctokit = Octokit.plugin(createPullRequest);
+
+  const octokit = new MyOctokit({
+    auth: TOKEN,
+  });
+
+  // Returns a normal Octokit PR response
+  // See https://octokit.github.io/rest.js/#octokit-routes-pulls-create
+  octokit
+    .createPullRequest({
+      owner: GITHUB_OWNER,
+      repo: GITHUB_REPO,
+      title: `Add nominee: ${values.name}`,
+      body:
+        "Automatic addition of a new nominee submitted through the online form available at https://digitalpublicgoods.net/submission",
+      base: GITHUB_BRANCH,
+      head:
+        `${values.name}`.replace(/ /g, "_") +
```

Comment:
For uniformity, I suggest we reuse the processed 'name' from L112 above (without the `.json` extension obviously)

Reply:
Resolved in commit a7c7df8.